AppSyncRealTimeClientTests/Connection/AppSyncSubscriptionConnectionTests.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  AppSyncSubscriptionConnectionTests.swift
+//  AppSyncSubscriptionClientTests
+//
+//  Created by Law, Michael on 2/19/20.
```

Comment:
Update with the doc that we use in other packages

Reply:
thanks, updating